ecms_base/modules/custom/ecms_ckeditor/ecms_ckeditor.ckeditor5.yml

```diff
@@ -0,0 +1,15 @@
+ecms_ckeditor_tableColumnResize:
+  ckeditor5:
+    plugins:
+      - table.TableColumnResize
```

Comment:
Is there any javascript that needs included?

Reply:
no.